### DISCUSSION TITLE
Outdated tutorial

### DISCUSSION THREAD

**Discussion by gozdii** [ourownstory/neural_prophet#890] [Feature Requests & Feedback]
In autoregression tutorial(https://neuralprophet.com/autoregression_yosemite_temps.html) you are using m.plot_last_forecast(forecast) but it seems like it was deleated

**Comment by noxan** [comment]
Thanks for pointing this out @gozdii - the function has been renamed to `plot_latest_forecast()` and we fixed the broken code with a deprecation warning - hope this solves the issue, also we'll update the tutorial to the new syntax.
